libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -101,28 +106,15 @@ export const AccordionItem = forwardRef<HTMLButtonElement, AccordionItemProps>(
       }
     }
 
-    const handleClose = () => {
-      expanded = false
-      setHeight(0)
-
-      if (expandedProp === undefined) {
-        setExpandedFallback(false)
-      }
-
-      if (typeof onToggle === 'function') {
-        onToggle(false)
-      }
-    }
-
-    const onHandleOpen = useCallback(handleOpen, [])
-
-    useEffect(() => {
-      if (startExpanded) {
-        onHandleOpen()
-      } else {
-        handleClose()
-      }
-    }, [onHandleOpen, startExpanded])
+    useEffect(
+      () => {
+        if (startExpanded && expandedProp == null) {
```

Comment:
```suggestion
        if (startExpanded && expandedProp === null) {
```

Reply:
Nope, this is checking if `expandedProp` is either `null` or `undefined`.

The options are either using `==` or going with fully expaneded double-check, like so:

```ts
        if (startExpanded && (expandedProp === null || expandedProp === undefined)) {
```

which is IMO excessively verbose.

Reply:
I guess this would be sufficient, since the type signature does not allow `null`.

```ts
        if (startExpanded && expandedProp === undefined) {
```

Reply:
I'm a fan of generally allowing `== null` and `!= null` to catch both undefined and null cases but I'm an old paranoid geezer. 😄

Reply:
Yeah, for me it's a habit to use `== null` almost like a shorthand for `=== undefined` and the fact it also captures any stray `null`s that might hypothetically float through (perhaps via slightly inaccurate upstream typings (often API related)) just feels like a minor bonus.